src/CircleGraph/CircleGraph.ts

```diff
@@ -110,12 +110,16 @@ export class CircleGraphPanel {
   }
 }
 
-function getWebviewOptions(extensionUri: vscode.Uri): vscode.WebviewOptions {
+function getWebviewOptions(extensionUri: vscode.Uri): vscode.WebviewOptions&
+    vscode.WebviewPanelOptions {
```

Comment:
Could you explain a bit about the reason of changing the return type?

Reply:
`createWebviewPanel` option is `options?: WebviewPanelOptions & WebviewOptions` type.
I added `WebviewOptions` type to use `WebviewPanelOptions.retainContextWhenHidden`